### PR TITLE
Tests for dimensiondata_load_balancer module

### PR DESCRIPTION
**Pull request by relaxdiego**
##### ISSUE TYPE

New Module Pull Request
##### COMPONENT NAME

dimensiondata/dimensiondata_load_balancer
##### ANSIBLE VERSION

```
ansible 2.2.0 (new_dimensiondata_load_balancer 86ac3fa51d) last updated 2016/09/28 22:42:41 (GMT -700)
  lib/ansible/modules/core: (detached HEAD db5cb54b23) last updated 2016/09/08 14:25:22 (GMT -700)
  lib/ansible/modules/extras: (new_dimensiondata_load_balancer aadef57314) last updated 2016/09/28 22:41:15 (GMT -700)
  config file = 
  configured module search path = Default w/o overrides
```
##### SUMMARY

These are the accompanying tests for https://github.com/ansible/ansible-modules-extras/pull/2982. Also note that any preceding commits that show changes in `module_utils/dimensiondata.py` do not belong to this PR but are, instead, part of PR #17604 which this is based off of.

### CODE REVIEW

**Comment by mattclay** [comment]
@relaxdiego One of the [test failures](https://app.shippable.com/runs/584f23963a097e0f00d803cf/1/console) is due to python 2.4 syntax errors:

```
2016-12-12 23:38:01 Compiling ./lib/ansible/module_utils/dimensiondata.py ...
2016-12-12 23:38:01   File "./lib/ansible/module_utils/dimensiondata.py", line 221
2016-12-12 23:38:01     except DimensionDataAPIException as e:
2016-12-12 23:38:01                                       ^
2016-12-12 23:38:01 SyntaxError: invalid syntax
```

Here's an example of how to handle exceptions for python 2.4 compatibility:

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/system/user.py#L317

File: test/utils/tox/requirements-py3.txt [hunk 1]
```diff
@@ -13,3 +13,4 @@ python3-memcached
 python-systemd
 botocore
 boto3
+apache-libcloud
```
Comment (mattclay) [inline test/utils/tox/requirements-py3.txt:16]: CI does not use this requirements file. Take a look at https://github.com/ansible/ansible/blob/devel/test/runner/requirements/units.txt for requirements for unit tests.

File: test/utils/tox/requirements.txt [hunk 2]
```diff
@@ -14,3 +14,4 @@ python-systemd
 pycrypto
 botocore
 boto3
+apache-libcloud
```
Comment (mattclay) [inline test/utils/tox/requirements.txt:17]: CI does not use this requirements file. Take a look at https://github.com/ansible/ansible/blob/devel/test/runner/requirements/units.txt for requirements for unit tests.

**Comment by mattclay** [comment]
@relaxdiego Unit tests for modules should no longer have a `core` or `extras` directory in their path. This was eliminated during the repo merge that occurred last week.

**Comment by tintoy** [comment]
@relaxdiego - is this PR still valid? I can try to pull these changes into another repo and then open a new PR to Ansible if that's the case.

**Comment by relaxdiego** [comment]
@tintoy sorry man but it's been a long time since I've looked at this code so I have no idea. :-) Your call if you want to keep it or not.

**Comment by tintoy** [comment]
How about you close your PR for now (you've still got the code so it's not in danger of being lost) and I can pull your commits across into the DDResearch repo when we actually get as far as the LB stuff :)

**Pull request closed without merging**